packages/table/README.md

```diff
@@ -893,3 +893,116 @@ For each table column you want to sort, use the `sortable` attribute in its resp
         });
     });
 </script>
+
+## Accessibility
+
+Tables can be created to allow the selection of single or multiple rows. In both cases, the checkbox in front of a row will have an `aria-label` that improves accessibility.
+
+The `<sp-table>` component is built to be accessible by default. The `aria-label` attribute is automatically assigned to the checkbox, providing accessibility support. The English default value for a row checkbox is 'Select'. The checkbox in the table header will have the value 'Select All'.
+
+Both strings can be customized by setting the `selectRowLabel` and `selectAllRowsLabel` attributes of the `<sp-table>` component. Providing selectRowLabel for the `<sp-table-row>` component will override the default value for that row. Providing selectAllRowsLabel for the `<sp-table>` component will override the default value for the row checkbox.
+
+### Combination of `selects`, `selectRowLabel` and `selectAllRowsLabel`
```

Comment:
Can `selects` be capitalized to be consistent with the table below? => `Selects`